packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -402,6 +403,10 @@ export default defineComponent({
     })
 
     const isPastingIntoSameFolder = computed(() => {
```

Comment:
The name of the `computed` is not good anymore with this change. Can you think of another one?

Reply:
Same for the user facing error message on line 424:
`return $gettext('You cannot paste resources into the same folder.')`

That's more like `You cannot cut&paste resource within the same folder.` or something like that.